### PR TITLE
`Minimum Safety Threshold` update

### PR DESCRIPTION
**Pull request by MikePlante1**
- Changes default of minimum safety threshold in Trio to `60` from `65` to match standard oref.
  - Standard oref sets threshold to `20 + ½ TargetGlucose`. Since 80 mg/dL is the lowest oref will use as a target, 60 mg/dL is the lowest oref will set the safety threshold to.
  - Fixes typo in Trio-oref that defaulted minimum threshold to 0.6, even though it was still raised to 60 later.
- Changes name of setting from `Threshold Setting (mg/dL)` to `Minimum Safety Threshold (mg/dL)`.
- Updates tooltip to improve clarity.
- Resets translations for setting name and tooltip

tooltip

### CODE REVIEW

File: trio-oref/lib/profile/index.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function defaults ( ) {
     , tddAdjBasal: false // Enable adjustment of basal based on the ratio of 24 h : 10 day average TDD
     , enableSMB_high_bg: false // enable SMBs when a high BG is detected, based on the high BG target (adjusted or profile)
     , enableSMB_high_bg_target: 110 // set the value enableSMB_high_bg will compare against to enable SMB. If BG > than this value, SMBs should enable.
-    , threshold_setting: 0.60 // Use a configurable threshold setting
```
Comment (JeremyStorring) [inline trio-oref/lib/profile/index.js:80]: Oh my - was this always supposed to be 60 instead of 0.60?

**Review by JeremyStorring — DISMISSED** [review]

**Comment by pheltzel** [comment]
LGTM!

File: trio-oref/lib/profile/index.js (hunk 1 above)
Comment (bjornoleh) [reply trio-oref/lib/profile/index.js:80]: Yes, a typo that we didn’t introduce, but which also made no difference due to minimum limits imposed elsewhere. Please see  https://github.com/nightscout/trio-oref/pull/26 for some more info :-)

**Comment by bjornoleh** [comment]
Minor suggestion for improvement (?), as it may not be clear what the “If you use mmol” part actually means:

Must be set between 60-120 mg/dL. If you use mmol/L, multiply by 18

-> Must be set between 60-120 mg/dL. To convert from mmol/L to mg/dL, please multiply by 18

**Comment by MikePlante1** [comment]
> -> Must be set between 60-120 mg/dL. To convert from mmol/L to mg/dL, please multiply by 18

Thanks. Just pushed that update.

**Review by bjornoleh — APPROVED** [review]

**Review by dnzxy — APPROVED** [review]
LGTM

**Comment by bjornoleh** [comment]
Merging with two approving reviews

**Pull request merged**